Doc/using/cmdline.rst

```diff
@@ -994,3 +994,73 @@ Debug-mode variables
    Need Python configured with the :option:`--with-trace-refs` build option.
 
    .. versionadded:: 3.11
+
+.. _using-on-finding-modules:
+
+Finding modules
+---------------
+
+A module search path is initialized when Python starts. This module search path
+may be accessed at :data:`sys.path`.
+
+The first entry in the module search path is the directory that contains the
+input script, if there is one. Otherwise, the first entry is the current
+directory, which is the case when executing the interactive shell, a ``-c``
+command, or ``-m`` module.
+
+On Windows a default Python installation adds the :file:`/Lib` directory and
+the :file:`/Lib/site-packages` directory to the module search path. These
+locations are relative to the :file:`python.exe` location. Standard modules
+are usually found in the Lib directory and third-party modules in the
+Lib/site-packages directory.
+
+On Unix the lib and lib/site-packages directories are added by looking for
+:file:`/{exec_prefix}/lib/pythonversion/site-packages` , :file:`/{exec_prefix}/lib/pythonversion/lib-dynload` ,
+:file:`/{prefix}/lib/pythonversion` and :file:`/{prefix}/lib/pythonversion/site-packages` .
+Often ``exec_prefix`` and ``prefix`` are the same and a common result is
+adding :file:`/usr/lib/pythonversion` and :file:`/usr/lib/pythonversion/site-packages`
+to the module search path. See :ref:`using-on-unix` for details.
```

Comment:
The "{prefix}" and "{exec_prefix}" values from `sys.prefix` and `sys.exec_prefix` should be absolute paths, so the leading "/" shouldn't be there.

For the standard library it's actually "{base_prefix}" and "{base_exec_prefix}". For site packages it's "{prefix}" and "{exec_prefix}", which get overridden by a virtual environment.

Reply:
And just to be clear, `getpath.py` does basically nothing for virtual environments apart from using the config to locate the base environment. It's `site.py` that does the actual search paths there.

Reply:
I was thinking, since virtual environments are important, that the description should be consistent with running in one. So use `base_prefix` and `base_exec_prefix` for the location of the standard library and system site packages. Without a virtual environment, they should be the same as `prefix` and `exec_prefix`.

The basic description doesn't need to distinguish how and where the search path gets defined, but maybe it's clearer to state that it applies to running without `-S`.